waiter/src/waiter/auth/spnego.clj

```diff
@@ -84,7 +85,8 @@
   (log/info "triggering 503 unavailable for spnego authentication")
   (counters/inc! (metrics/waiter-counter "core" "response-status" "503"))
   (meters/mark! (metrics/waiter-meter "core" "response-status-rate" "503"))
-  (-> {:message "Too many Kerberos authentication requests"
+  (-> {:details {:error-class "waiter.KerberosQueueLength"}
```

Comment:
Should these error classes be defined constants. For example, the "http-503-service-unavailable" is defined in status_codes.clj?

Reply:
Good idea, will change.

Reply:
The status codes file is for well-defined status codes, so didn't put these in that file.
Instead, I introduced constants in each file for the error classes referenced in each file.